src/raft.c

```diff
@@ -355,24 +352,21 @@ static int answer_HEARTBEAT (
 	set_leader(tk, leader);
 
 	/* Ack the heartbeat (we comply). */
-	return send_msg(OP_ACK, tk, sender, msg);
+	return send_msg(conf, OP_ACK, tk, sender, msg);
 }
 
 
-static int process_UPDATE (
-		struct ticket_config *tk,
-		struct booth_site *sender,
-		struct booth_site *leader,
-		struct boothc_ticket_msg *msg
-	       )
+static int process_UPDATE (struct booth_config *conf, struct ticket_config *tk,
```

Comment:
I think it make sense to remove space between `process_UPDATE` and `{` similarly as elsewhere (`answer_HEARTBEAT`, `process_REVOKE`)